plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -483,4 +485,30 @@ public IcebergConfig setMetadataCacheEnabled(boolean metadataCacheEnabled)
         this.metadataCacheEnabled = metadataCacheEnabled;
         return this;
     }
+
+    @Config("iceberg.object-store.enabled")
+    @ConfigDescription("Enable the Iceberg object store file layout")
+    public IcebergConfig setObjectStoreEnabled(boolean objectStoreEnabled)
```

Comment:
I'm trying to get familiar with the functionality.

I see the Iceberg commit

https://github.com/apache/iceberg/commit/b6fa5494c28f5e99a52af1ce3963352b4dc1aaf5

I've seen a very brief description on https://iceberg.apache.org/docs/latest/configuration/#write-properties


write.object-storage.enabled | false | Enables the object storage location provider that adds a hash component to file paths
-- | -- | --


Can you please explain in the PR description what does this feature do?
Maybe even add a file listing to showcase how the file names get affected.

Reply:
Updated and provided a link to Iceberg's more descriptive documentation on the feature